src/main/java/org/gitlab4j/api/DeployKeysApi.java

```diff
@@ -192,6 +192,24 @@ public DeployKey addDeployKey(Object projectIdOrPath, String title, String key,
         return (response.readEntity(DeployKey.class));
     }
 
+    /**
+     * Updates an existing project deploy key.
+     *
+     * <pre><code>GitLab Endpoint: PUT /projects/:id/deploy_keys/:key_id</code></pre>
+     *
+     * @param projectIdOrPath the project in the form of an Integer(ID), String(path), or Project instance
+     * @param deployKey a Deploy key instance with updated properties
+     * @return an updated DeployKey instance
+     * @throws GitLabApiException if any exception occurs
+     */
+    public DeployKey updateDeployKey(Object projectIdOrPath, DeployKey deployKey) throws GitLabApiException {
```

Comment:
@adamMo Since only ```title``` and ```canPush``` properties can be updated and both are optional I would prefer  thta this method take parameters as follows:
```
public DeployKey updateDeployKey(Object projectIdOrPath, Integer deployKeyId, String title, Boolean canPush)
```
Sending the entire ```DeployKey``` instance will be sending information that is not needed.  If you create a ```DeployKey``` instance and populate it with just the values of ```title``` and ```canPush``` which either can be null will be better.

Reply:
I made it in two methods now, let me know if it's acceptable for you.

Reply:
@adamMo Thanks for the update, however I'd like to remove the method that takes a ```DeployKey``` instance.  The pattern for this library is to pass objects in only when there are many parameters.

Reply:
Ok, it's removed.